GCS/probe0/probe0.go

```diff
@@ -0,0 +1,143 @@
+package probe0
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"io"
+	"os"
+	"time"
+	"cloud.google.com/go/storage"
```

Comment:
Nit: add an empty line above this to create two import groups:

1) Standard library packages
2) Other packages